e2e/tests/root.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Root Namespace
+
+  Scenario: User Needs Help
+    Given the user has a pocket client 
+    When the user runs the command "help"
+    Then the user should be able to see standard output containing "Available Commands"
+    And the pocket client should have exited without error
```

Comment:
This test format was copied form the v0 tests to show feature parity.

Reply:
Great starting point, especially because `help` is appropriate everywhere.

I'd suggest not going for feature parity with v0 for the real business logic. We can make it better, simpler and more effective :)